barcodes_generator_abstract/models/barcode_generate_mixin.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2014-TODAY GRAP (http://www.grap.coop)
+# Copyright (C) 2016-TODAY La Louve (http://www.lalouve.net)
+# Copyright 2017 LasLabs Inc.
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import _, api, exceptions, fields, models
+
+from .barcode_rule import _GENERATE_TYPE
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import barcode
+except ImportError:
+    _logger.debug("Cannot import 'python-barcode' python library.")
+    barcode = None
```

Comment:
```suggestion
import barcode
```

Probably `ImportError` it's impossible because if defined in manifest file.